packages/dear-imgui/cpp/tests/hello_test.cc

```
@@ -0,0 +1,9 @@
+#include <gtest/gtest.h>
+
+// Demonstrate some basic assertions.
+TEST(HelloTest, BasicAssertions) {
```

cppcheck

[style] The function 'TEST' is never used.

(unusedFunction)